CHANGELOG.rst

```diff
@@ -25,18 +76,21 @@ This project adheres to `Semantic Versioning <https://semver.org/>`_.
 
 * Adjust agreement text in offer template and move table header below section/product category title in offer html template (`#606 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/606>`_)
 
+* Move table header below section/product category title in offer html template (`#604 <https://github.com/qbicsoftware/offer-manager-2-portlet/pull/604>`_)
+
```

Comment:
why is this added?

Reply:
this should be part of the current release

Reply:
it is, you are in the section of the 1.0.0-release

Reply:
no its in the section oft the release candidate. This should have been included already last release and not be introduced now

Reply:
maybe so forgot to add it then